scania-digital-design-system/sdds
# fix(breadcrumb): remove link props and add default slot

**Describe pull-request**  
Opened default slot for breadcrumb and removed "link-props". This to enable the user to slot in any sort of link.

With this API change the user will slot in a <a> or other link element in sdds-breadcrumb-item, much like the tabs and our sdds-link.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Breadcrumb -> WebComponent
3. Check that the styling is correct.

**Suggested test steps**
